src/index.html

```diff
@@ -10,7 +10,40 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form
+        action="/"
+        method="get"
+        data-qa="big"
+      >
+        <label for="big-search">
```

Comment:
If you add `for` attribute (which is good), you need to add corresponding `id` to the input.